### PR TITLE
(PE-34578) add bouncycastle with jdk8 and up support

### PR DESCRIPTION
**Pull request by jonathannewman**
See individual commit descriptions.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -142,7 +142,12 @@
                          [org.bouncycastle/bc-fips "1.0.2.1"]
                          [org.bouncycastle/bctls-fips "1.0.11.4"]
                          [org.bouncycastle/bcpkix-jdk15on "1.70"]
-                         [org.bouncycastle/bctls-jdk15on "1.70"]]
+                         [org.bouncycastle/bctls-jdk15on "1.70"]
```
Comment (nmburgan) [inline project.clj:145]: bcpkix and bctls are also jdk18on

**Review by justinstoller — APPROVED** [review]

File: project.clj (hunk 1 above)
Comment (steveax) [inline project.clj:145]: This should be jdk18on and 1.7.2, yes?

**Review by steveax — DISMISSED** [review]

File: project.clj (hunk 1 above)
Comment (steveax) [reply project.clj:145]: ahh, nevermind, I see now

**Review by steveax — APPROVED** [review]
👍

**Review by nmburgan — APPROVED** [review]

**Pull request merged**